x-pack/filebeat/input/http_endpoint/handler.go

```diff
@@ -110,15 +114,15 @@ func decodeJSON(body io.Reader) (objs []common.MapStr, rawMessages []json.RawMes
 	for decoder.More() {
 		var raw json.RawMessage
 		if err := decoder.Decode(&raw); err != nil {
-			if err == io.EOF {
+			if errors.Is(err, io.EOF) {
```

Comment:
This will never be a wrapped error. I think that this linter's advice is poorly founded and is ironic given the length gone to to avoid allocs here.